nanome/beta/nanome_sdk/plugin_server.py

```diff
@@ -131,9 +131,12 @@ async def route_bytes(self, received_bytes):
 
         elif packet_type == PacketTypes.client_disconnection:
             logger.info(f"Disconnecting Session {session_id}.")
-            popen = self._sessions[session_id]
-            popen.kill()
-            del self._sessions[session_id]
+            if session_id in self._sessions.keys():
```

Comment:
```suggestion
            if session_id in self._sessions:
```
`in` operator checks if key in `dict`